README.md

```diff
@@ -125,6 +129,62 @@ Kafka. This value cannot be set, only read.
 
 Aiven ID format when importing existing resource: name of the project as is.
 
+### Resource Account
+```
+resource "aiven_account" "foo" {
```

Comment:
myaccount or somesuch might more readable sample value than foo. Sample for other similar cases.